www/assets/js/webauthn.js

```diff
@@ -43,7 +43,7 @@ function registrationButtonClick() {
 function authButtonClick() {
     navigator.credentials.get(publicKeyCredentialRequestOptions)
     .then((cred) => {
-        console.log('NEW CREDENTIAL', cred);
+        console.log('AUTH', cred);
```

Comment:
As a general rule, I think we should avoid logging anything to the console, at least for production.